src/main/webapp/js/common.js

```diff
@@ -959,9 +960,11 @@ function hideSingleCollapse(e) {
  *  Bootstrap modals"
  */
 var BootboxWrapper = {
-    DEFAULT_OK_TEXT: 'OK',
+    DEFAULT_OK_TEXT: 'Ok',
```

Comment:
You do not need to change this. "OK" is commonly used in all variants "OK", "Ok", "ok".